src/views/Migrate.vue

```diff
@@ -19,28 +41,45 @@ import { Component, Vue } from 'vue-property-decorator';
 import { AccountInfo } from '@/lib/AccountInfo';
 import { WalletStore } from '@/lib/WalletStore';
 import { WalletInfo, WalletType } from '@/lib/WalletInfo';
-import { SmallPage } from '@nimiq/vue-components';
+import { SmallPage, PageHeader, PageBody, PageFooter, AccountList } from '@nimiq/vue-components';
 import Network from '@/components/Network.vue';
 import Loader from '@/components/Loader.vue';
-import { KeyguardClient } from '@nimiq/keyguard-client';
+import KeyguardClient from '@nimiq/keyguard-client';
 import { ACCOUNT_DEFAULT_LABEL_LEGACY } from '@/lib/Constants';
 import { ContractInfo } from '@/lib/ContractInfo';
+import { Static } from '@/lib/StaticStore';
+import { SimpleRequest } from '@/lib/PublicRequestTypes';
 
-@Component({components: {SmallPage, Loader, Network}})
+@Component({components: {SmallPage, PageHeader, PageBody, PageFooter, AccountList, Loader, Network}})
 export default class Migrate extends Vue {
+    @Static private request!: SimpleRequest;
+
     private title: string = 'Migrating your accounts';
     private status: string = 'Connecting to Keyguard...';
     private state: Loader.State = Loader.State.LOADING;
     private message: string = '';
 
-    private keyguardClient?: KeyguardClient;
+    private legacyAccounts: AccountInfo[] = [];
+    private isMigrating: boolean = false;
+
+    public async created() {
+        const legacyKeys = await this.$rpc.keyguardClient.listLegacyAccounts();
+        this.legacyAccounts = legacyKeys.map((key) => this.legacyKeyInfoObject2AccountInfo(key));
```

Comment:
As the `legacyKeyInfoObject2AccountInfo` method is only used here, I would inline it.